### DISCUSSION TITLE
How to request configuration options for an existing rule?

### DISCUSSION THREAD

**Discussion by espertus** [pmd/pmd#4568] [Q&A] [answered]
I have started using PMD and would prefer that `CommentRequired` not flag private classes and members. As far as I can tell, the only ways to keep that rule from triggering on private fields (for example) are :

- explicitly suppressing warnings
- setting the property `fieldCommentRequirement` to `ignored`

I would like to suggest that configuration options be added so went to file an issue, which required me to choose among these options:

- Rule violation (false-positive)
- Rule violation (false-negative)
- New Rule
- Feature request
- Bug report

Which of these should I choose, or is the issue tracker the wrong place entirely?

Thanks!

**Comment by oowekyala** [comment, marked as answer]
Hi, a "false positive" report is appropriate in this case. While this is technically a feature request, the FP template is more useful to us IMO because it asks for more information, like a code sample and a link to the rule.
